app/assets/stylesheets/components/about.scss

```diff
@@ -0,0 +1,198 @@
+            text-align: center;
+            color: $primary;
+        }
+        .row {
+            width: 70%;
+        }
+        .leftcol {
+            width: 40%;
+            img {
+                width: 200px;
+                margin-right: 20px;
+                float: right;
+            }
+        }
+        .rightcol {
+            width: 55%;
+            div.speech-bubble {
+                text-align: center;
+                background: $light-grey;
+                padding: 20px;
+                width: 320px;
+                
+                position: relative;
+                z-index: 1;
+                margin: auto;
+            }
+                
+            div.speech-bubble:after {
+                content: "";
+                position: absolute;
+                width: 0;
+                height: 0;
+                transform: rotate(-45deg);
+                left: -25px;
+                top: 80px;
+                border: 25px solid;
+                border-color: $light-grey transparent transparent $light-grey;
+            }
+        }
+        
+        .cta {
+            text-decoration: none;
+            background-color: $primary;
+            color: #ffffff;
+            display: block;
+            text-align: center;
+            padding: 0 1.5em 0 1.5em;
+            width: 200px;
+            height: 60px;
+            border-radius: 3px;
+            line-height: 60px;
+            margin: 70px auto 70px;
+            box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.5);
+            font-weight: bold;
+            letter-spacing: 0.2px;
+            font-size: 18px;
+        }
+    }
+    .contribute {
+        text-align: center;
+        color: #999;
+        background: linear-gradient(white, $light-grey);
+        .cta {
+            text-decoration: none;
+            background-color: $button-primary;
+            color: #ffffff;
+            display: block;
+            width: 350px;
+            text-align: center;
+            height: 60px;
+            border-radius: 3px;
+            line-height: 60px;
+            margin: 70px auto 0;
+            box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.5);
+            font-weight: bold;
+            letter-spacing: 0.2px;
+            font-size: 18px;
+        }
+    }
+    
+    .goldbar {
+        border-top: solid $primary 2px;
+        position: relative;
+    }
+    
+    .goldbar:before {
+        content: "";
+        width: 20px;
+        height: 20px;
+        border-radius: 50%;
+        top: -10px;
+        left: -60px;
+        background: $primary;
+        display: block;
+        position: relative;
+    }
+    
+}
+
+@media (max-width: 768px) {
+   #about {
+    .mission {
+      img {
+        float: none;
+        display: block;
+        margin: auto;
+      }
+      .leftcol {
+          width: 100%;
+      }
+      .rightcol {
+          width: 100%;
+      }
+    }
+    .contact {
+        .leftcol {
+            width: 100%;
```

Comment:
This is to make the columns stack on the mobile view only...they are different % on main css